uportal-war/src/main/data/quickstart_entities/fragment-layout/guest-lo.fragment-layout.xml

```diff
@@ -36,6 +36,7 @@
             <channel fname="personalization-gallery" unremovable="false" hidden="false" immutable="false" ID="n42"/>
         </folder>
         <folder ID="s100" hidden="false" immutable="true" name="Pre Header folder" type="pre-header" unremovable="true">
+            <channel fname="portal-menu-toggle" unremovable="false" hidden="false" immutable="false" ID="n9"/>
```

Comment:
No problem!  The notes are for myself as much as anyone... trying (finally) to understand and merge this PR.  There are some conflicts, so I'm obligated to do a bit of hand-tweaking anyway.